examples/forest_fire.jl

```diff
@@ -20,21 +20,23 @@
 # ## Defining the core structures
 
 # We start by defining the agent type
-using Agents, Random
+using Agents, Random, AgentsPlots, Plots
+pyplot() # hide
 
 mutable struct Tree <: AbstractAgent
     id::Int
     pos::Tuple{Int,Int}
     status::Bool  # true is green and false is burning
 end
+nothing # hide

```

Comment:
I tried to step the matplotlib install from spilling its output here, but this didn't fix it. Any ideas?
https://juliadynamics.github.io/Agents.jl/previews/PR245/examples/forest_fire/

Reply:
Just plot something random before `makedocs` command I think. (with `pyplot()`)

Reply:
Great, that worked!